Objective-C/CBLReplicatorConfiguration.m

```diff
@@ -205,6 +205,8 @@ - (NSDictionary*) effectiveOptions {
     NSString* uniqueID = $castIf(CBLMessageEndpoint, _target).uid;
     if (uniqueID)
         options[@kC4ReplicatorOptionRemoteDBUniqueID] = uniqueID;
+    
+    options[@kC4ReplicatorOptionOnlySelfSignedServerCert] = @(_acceptOnlySelfSignedServerCertificate);
```

Comment:
pass the acceptOnly flag to the lite core

Reply:
See my comment.